arbnode/batch_poster.go

```diff
@@ -615,7 +615,7 @@ func (b *BatchPoster) ParentChainIsUsingEIP7623(ctx context.Context, latestHeade
 	if diffIsClose(gas1, gas2, 14, 18) {
 		// targetDiff is 16
 		parentChainIsUsingEIP7623 = false
-	} else if diffIsClose(gas1, gas2, 36, 44) {
+	} else if diffIsClose(gas1, gas2, 36, 200) {
 		// targetDiff is 40
```

Comment:
Comment should be updated (future PR)

Reply:
Actually, that is not necessary for this change. It was only temporarily in place. Reverted through additional commits.